tests/30rooms/04messages.pl

```diff
@@ -22,27 +22,33 @@
    do => sub {
       my ( $senduser, $local_user, $room_id ) = @_;
 
-      matrix_send_room_message( $senduser, $room_id,
-         content => { msgtype => $msgtype, body => $msgbody },
+      Future->needs_all(
+         matrix_sync( $senduser ),
```

Comment:
why do we have these extra syncs now?

Reply:
Turns out that this was because the test after it was relying on there being a sync token. That test was buggered anyway so I've changed it to not rely on that.